index.js

```diff
@@ -731,9 +735,33 @@ function buildCode (schema, code, laterCode, name, externalSchema, fullSchema) {
       `
     }
   })
+
+  if (required && required.length > 0) {
+    code += 'var required = ['
+    for (var i = 0; i < required.length; i++) {
+      if (i > 0) {
+        code += ','
+      }
+      code += `'${required[i]}'`
```

Comment:
This should be properly escaped.

Reply:
I escaped using `$asString` function, is this the correct way to escape?